sdk/core/Azure.Core.Experimental/src/MutableJsonDocument.cs

```diff
@@ -45,15 +45,14 @@ public MutableJsonElement RootElement
         /// <summary>
         /// Writes the document to the provided stream as a JSON value.
         /// </summary>
-        /// <param name="stream"></param>
-        /// <param name="format"></param>
-        /// <exception cref="ArgumentOutOfRangeException"></exception>
+        /// <param name="stream">The stream to write the document to.</param>
+        /// <param name="format">The format to write the output in.</param>
+        /// <exception cref="FormatException">Thrown if an unsupported value is passed for format.</exception>
         public void WriteTo(Stream stream, StandardFormat format = default)
         {
-            // this is so we can add JSON Patch in the future
-            if (format != default)
+            if (format != default || format.Symbol != 'J')
             {
-                throw new ArgumentOutOfRangeException(nameof(format));
+                throw new FormatException($"Unsupported format {format.Symbol}. Supported formats are: 'J' - JSON.");
```

Comment:
I'd at least document the formats in the doc comments like .NET does it using the list or table elements (I forget which) so developers know up front. Putting them in the exception is not something I've seen elsewhere.

Reply:
Do you have an example in refdocs that you like?  The [String.Format() method](https://learn.microsoft.com/en-us/dotnet/api/system.string.format?view=net-8.0#system-string-format(system-string-system-object-system-object)) links out to other docs

Reply:
I can add `<remarks>` for it - that looks standard in dotnet/runtime

Reply:
`DateTimeOffset.ToString(string)` has a good example in both "Examples" and "Remarks", where remarks uses the table format I was talking about: https://learn.microsoft.com/en-us/dotnet/api/system.datetimeoffset.tostring?view=net-8.0#system-datetimeoffset-tostring(system-string)

Something like,

```C#
/// <remarks>
/// ...
/// <list type="table">
///   <listheader>
///     <term>Format</term>
///     <description>Description></description>
///   </listheader>
///   <item>
///     <term>J</term>
///     <description>JSON</description>
///   </item>
/// </list>
```

Reply:
Oh no, I merged too fast to take this suggestion!  I'll include it in a later PR -- thanks for it, @heaths!

Reply:
Looking at this now - I think it will make sense to use this format for the remarks when we have more than one supported format, i.e. when we add support for JSON merge patch.  I'll track the request with that feature issue.